microk8s-resources/wrappers/openssl.wrapper

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+export PATH="$SNAP/usr/sbin:$SNAP/usr/bin:$SNAP/sbin:$SNAP/bin:$PATH"
+ARCH="$($SNAP/bin/uname -m)"
+export LD_LIBRARY_PATH="$SNAP/lib:$SNAP/usr/lib:$SNAP/lib/$ARCH-linux-gnu:$SNAP/usr/lib/$ARCH-linux-gnu"
```

Comment:
Can you please rebase? These should not be needed any longer with the recent changes to snapcraft.yaml